docker-compose.yml

```diff
@@ -19,4 +19,10 @@ services:
     ports:
       - "3000:3000"
     depends_on:
-      - db
\ No newline at end of file
+      - db
+  swagger:
+    image: swaggerapi/swagger-ui:v3.0.5
+    ports:
+      - 8080:8080
+    environment:
+      API_URL: https://raw.githubusercontent.com/aerogear/mobile-security-service/master/api/swagger.yaml
```

Comment:
Could it be changed to get the data from the local file in the dev env? In this way, the UI will reflect the changes performed locally.

Reply:
I don't believe so. This came up before and we came to the conclusion that it could not be done.

Reply:
Swagger needs the file to be served over http from what I found.

Reply:
However, is not possible to retrieve the file by something as localhost:3000/api ...?

Reply:
If there is a web server running locally, that is indeed possible.